gatsby/config-options/constants.ts

```diff
@@ -1,10 +1,11 @@
-import { buildGatsbyCloudPreviewUrl } from '../util/build-gatsby-cloud-preview-url';
+import { buildNetlifyPreviewUrl } from '../util/build-netlify-preview-url';
 export const RSS_FEED_URL = '/blog/rss.xml';
 export const DEFAULT_META_IMAGE_URL_PATH = '/sy-share-image.jpg';
-export const GATSBY_SITE_PREFIX = process.env.GATSBY_SITE_PREFIX || '';
-export const BRANCH_PREVIEW_URL = buildGatsbyCloudPreviewUrl({
-  prefix: GATSBY_SITE_PREFIX,
+export const NETLIFY_DOMAIN_NAME = process.env.GATSBY_NETLIFY_DOMAIN_NAME || '';
+export const BRANCH_PREVIEW_URL = buildNetlifyPreviewUrl({
```

Comment:
the preview being null is mostly the case for non-preview environments. It will then use `GATBSY_BASE_URL` (following line). Is this something we create or given by netlify? Asking to make sure this will work when we merge it, because right now I can only verify the preview part of this PR and I fear it's not working later on production. Do you have confidence that this will work then? Thanks!

Reply:
The `GATSBY_BASE_URL` (and also the `GATSBY_PREFIX` which I changed to `GATSBY_NETLIFY_DOMAIN_NAME`) was part of our environment configuration on gatsby cloud and got lost on the migration to netlify. I readded it in the netlify settings. Hence, it currently works again on production. This PR only fixes the feature branch BASE_URL. for the main branch the BRANCH_PREVIEW_URL will still be "null" and the `GATSBY_BASE_URL` will be used. And since that works currently on production, I am confident, that it will still work once this PR is merged. But I will monitor it.